command/format.go

```diff
@@ -71,6 +72,14 @@ func outputFormatTable(ui cli.Ui, s *api.Secret, whitespace bool) int {
 		input = append(input, fmt.Sprintf("%s %s %v", k, config.Delim, v))
 	}
 
+	if len(s.Warnings) != 0 {
+		input = append(input, "")
+		input = append(input, "The following warnings were generated:")
```

Comment:
"... were returned from the Vault server"

I think "were generated" might be too broad and people may think it's an error from their CLI or operating system or something.